x-pack/plugins/enterprise_search/server/plugin.ts

```diff
@@ -309,7 +322,30 @@ export class EnterpriseSearchPlugin implements Plugin {
     }
   }
 
-  public start() {}
+  public start(core: CoreStart) {
+    /**
+     * Create exposed plugin services
+     */
+    if (!this.logger) {
+      throw new Error('Enterprise Search logger has not been initialized');
+    }
+    if (!this.clusterClient) {
+      this.clusterClient = core.elasticsearch.client;
+    }
+    if (!this.connectorsService) {
+      this.connectorsService = new ConnectorsService({
```

Comment:
Can this fail?

Reply:
Not really, but I'll add a try/catch so we don't crash Kibana accidentally.